starfish/intensity_table.py

```diff
@@ -307,32 +298,15 @@ def from_image_stack(
         y = np.arange(ymin, ymax)
         x = np.arange(xmin, xmax)
 
-        pixel_coordinates = pd.DataFrame(
+        attribute_data = pd.DataFrame(
             data=np.array(list(product(z, y, x))),
             columns=['z', 'y', 'x']
         )
-        pixel_coordinates[Features.SPOT_RADIUS] = np.full(
-            pixel_coordinates.shape[0], fill_value=np.nan)
-
-        image_size = cropped_data.shape[:3]
-
-        return IntensityTable.from_spot_data(intensity_data, pixel_coordinates, image_size)
+        attribute_data[Features.SPOT_RADIUS] = np.full(attribute_data.shape[0], fill_value=0.5)
```

Comment:
Leave a comment about these == pixels, therefore diameter should be easily calculated from the diameter. Think about area of 1 pixel and how to calculate this.

Reply:
There is a really interesting question about how we should calculate radius vs area.

Reply:
Added note to #380